lib/ansible/modules/cloud/amazon/ecs_task.py

```diff
@@ -56,10 +56,30 @@
         required: False
     network_configuration:
         description:
-          - network configuration of the service. Only applicable for task definitions created with C(awsvpc) I(network_mode).
-          - I(network_configuration) has two keys, I(subnets), a list of subnet IDs to which the task is attached and I(security_groups),
-            a list of group names or group IDs for the task
+          - Network configuration of the service. Only applicable for task definitions created with I(network_mode=awsvpc).
+          - I(assign_public_ip) requires botocore >= 1.8.4
+        required: False
         version_added: 2.6
+        type: dict
+        suboptions:
+          subnets:
+            description:
+              - A list of subnet IDs to associate with the task
```

Comment:
```suggestion
              - A list of subnet IDs to associate with the task.
```

Please see https://docs.ansible.com/ansible/latest/dev_guide/developing_modules_documenting.html#documentation-block

descriptions (but not short_descriptions) should be full sentences, including a full stop at the end.